Psychtoolbox/PsychCal/MeasureDpi.m

```diff
@@ -88,7 +127,7 @@
     Screen('Flip',window);
     oldButton=0;
     while 1
-        [x,y,button]=GetMouse;
+        [x,~,button]=GetMouse;
```

Comment:
One interesting catch here is that distances when measured on a HiDPI / Retina display might not be what one expects, due to some rescaling by the OS or PTB itself done for Retina style displays, especially on macOS or Windows 10.
Not sure. Might be good to provide GetMouse(theScreen) or GetMouse(window) to help GetMouse to understand to which display this refers. Other than that, experimentation with different OS'es and HiDPI displays...